<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResUpAndDownLoads</class>
 <widget class="QDialog" name="ResUpAndDownLoads">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;Aparajita&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_COM"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="comboBox_BAUD">
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pushButton_UART">
        <property name="text">
         <string>打开串口</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>37</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>项目名：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_inItem">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_IN">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="7" column="0" colspan="5">
           <widget class="QLabel" name="label_Show">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="5">
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_OUT">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioDownAndFile">
            <property name="text">
             <string>下载且保存</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QRadioButton" name="radioOnlyHardBtn">
            <property name="text">
             <string>仅下载到小板</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>项目名：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="lineEdit_OUTItem"/>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QRadioButton" name="radioOnlyFileBtn">
            <property name="text">
             <string>仅保存到文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
